x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -28510,7 +28510,6 @@
     "xpack.uptime.emptyStateError.notFoundPage": "ページが見つかりません",
     "xpack.uptime.emptyStateError.title": "エラー",
     "xpack.uptime.enableAlert.editAlert": "アラートを編集",
-    "xpack.uptime.featureRegistry.uptimeFeatureName": "アップタイム",
```

Comment:
See this removed, but don't see a corresponding synthetics replacement added.

Reply:
new translation will be generated for new keys auto.